### PR TITLE
V.3 changes

### PR DESCRIPTION
**Pull request by sharma-dark-knight**

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
 Package.describe({
   name: "froala:editor",
   summary: "A beautiful jQuery WYSIWYG HTML rich text editor.",
```
Comment (dheerajaccolite) [inline package.js]: Remove Jquery. Write. Javascript WYSIWYG

File: package.js [hunk 2]
```diff
@@ -78,7 +78,6 @@ Package.onUse(function(api) {
     // 3rd Party
     "assets/js/third_party/embedly.min.js",
     "assets/js/third_party/font_awesome.min.js",
-    "assets/js/third_party/image_aviary.min.js",
```
Comment (dheerajaccolite) [inline package.js:81]: why you have removed image_aviary???

Comment (sharma-dark-knight) [reply package.js:81]: It is not there in dist files shared by stefan. So was giving compilation error.

File: package.js (hunk 1 above)
Comment (sharma-dark-knight) [reply package.js]: Done

File: package.js (hunk 2 above)
Comment (dheerajaccolite) [reply package.js:81]: dist was just for testing.  cant change the framework code because of that.

Comment (dheerajaccolite) [reply package.js:81]: why its not part of our dist. Check.?

**Pull request merged**